docs/section12/Rule12-2.md

```diff
@@ -1,36 +1,40 @@
+- For each space in the baseline RMD: `for space_b in B_RMD.spaces:`  
+  - Get the lighting space type: `space_type_b = space_b.space_type`
+  - For each miscellaneous equipment load in the space: `for misc_equip_b in space_b.miscellaneous_equipment:`
+    - Get the corresponding miscellaneous equipment load from the proposed RMD: `misc_equip_p = match_data_element(P_RMD, MiscellaneousEquipment, misc_equip_b.id)`
+    - Get the baseline miscellaneous equipment load schedule: `misc_equip_schedule_b = misc_equip_b.multiplier_schedule`
+    - Get the proposed miscellaneous equipment load schedule: `misc_equip_schedule_p = misc_equip_p.multiplier_schedule`
+    - Get the baseline automatic receptacle control: `auto_receptacle_control_b = misc_equip_b.has_automatic_control`
+    - Get the proposed automatic receptacle control: `auto_receptacle_control_p = misc_equip_p.has_automatic_control`
+    - If it is a leap year, set the schedule comparison mask to 8784 hours, else set it to 8760 hours: `mask_schedule = [1] * 8784 if is_leap_year else [1] * 8760`
+    - Compare the baseline miscellaneous equipment load schedule to the proposed miscellaneous equipment load schedule: `comparison_data = compare_schedules(misc_equip_schedule_b, misc_equip_schedule_p, mask_schedule)`
+    
+**Rule Assertion:**  
+Case 1: If the baseline and proposed miscelleaneous equipment schedules match for all hours: PASS `if comparison_data['total_hours_matched'] == len(misc_equip_schedule_b == len(misc_equip_schedule_p): PASS`  
+Case 2: Elif the proposed automatic receptacle control was not specified: UNDETERMINED `elif auto_receptacle_control_p == Null: UNDETERMINED`  
+Case 3: Elif the baseline and proposed miscelleaneous equipment both have automatic controls, but schedules have unequal equivalent full load hours: FAIL and raise_message `elif auto_receptacle_control_p and auto_receptacle_control_b and comparison_data["eflh_difference"] != 0: FAIL and raise_message = "The baseline miscellaneous equipment schedule has automatic receptacle controls indicating that there is an applicable requirement for automatic controls for the space in Section 8.4.1. Miscellaneous equipment schedules may only differ when the proposed design has automatic receptacle controls and there are no applicable requirements in Section 8.4.1 for the space."`  
+Case 4: Elif the proposed miscellaneous equipment schedule has fewer equivalent full load hours than the baseline miscellaneous equipment schedule, the proposed has automatic receptacle control, and the space type is not expected to have receptacle control requirements in Section 8.4.1 : PASS `elif comparison_data["eflh_difference"] > 0 and auto_receptacle_controls_p and space_type_b not in EXPECTED_RECEPTACLE_CONTROL_SPACE_TYPES: PASS`  
```

Comment:
I wonder if we should add a schema element for RC = percentage of all controlled receptacles because then we could figure out if the schedules have been modeled correctly. I know this is not a model input but, from the dscusions I have been apart of, the RPD generator applications will have fields for additional inputs like this.